unifiednlp-base/src/main/java/org/microg/tools/selfcheck/NlpOsCompatChecks.java

```diff
@@ -47,7 +48,7 @@ public void doChecks(Context context, ResultCollector collector) {
     }
 
     private boolean checkSystemIsSupported(Context context, ResultCollector collector) {
-        boolean isSupported = (SDK_INT >= KITKAT && SDK_INT <= M);
+        boolean isSupported = (SDK_INT >= KITKAT && SDK_INT <= N_MR1);
```

Comment:
OK, but I'm not sure what could be the solution. It works for me on Android 7.1. If some specific version does not work, UnifiedNLP cannot say that the version is not supported. The version is supported but maybe some vendor specific extensions causes UnifiedNLP does not work. But intent driven API should work on all versions.